{{cookiecutter.repo_name}}/devtools/conda-envs/test_env.yaml

```diff
@@ -8,8 +8,13 @@ dependencies:
     # Testing
   - pytest
   - pytest-cov
-  - codecov
+{% if cookiecutter.dependency_source == 'Prefer default anaconda channel with pip fallback' %}
+  # Pip-only installs
+  - pip:
+    - codecov
+{% else %}  - codecov
```

Comment:
Can you check the formatting. Should be able to put it on another line to make this less confusing. Otherwise LGTM.

Reply:
There is an extra newline if not on the same line, ie the yml file looks like the following when I build a project on my computer

```
  - pytest
  - pytest-cov

  - codecov
```



see - http://jinja.pocoo.org/docs/dev/templates/#whitespace-control

Not sure if it's being handled differently in other parts, or if it just hasn't mattered.

Reply:
Hmm, thats a little odd. Oh well, in we go.